cloudbuild.yaml

```diff
@@ -89,8 +90,8 @@ steps:
     -pr-number=$_PR_NUMBER
     -skipped-validators=confd,yanglint
     -extra-pyang-versions=2.2.1
-    #-compat-report=yanglint
     -branch=$BRANCH_NAME
+    -compat-report=pyangbind
```

Comment:
Are we now making pyangbind non-blocking?

Reply:
There is an error in the BFD model: https://gist.github.com/OpenConfigBot/f2158a52bc907ed4e690582d47f2af61#gistcomment-4628801

I want to make it unblocking until we resolve it.

Reply:
Opened an issue in pyangbind: https://github.com/robshakir/pyangbind/issues/302